frontend/src/scenes/trends/ActionFilter/ActionFilterRow.js

```diff
@@ -111,36 +207,59 @@ export function ActionFilterRow({ logic, filter, index, hideMathSelector }) {
 }
 
 function MathSelector(props) {
-    let items = ['Total', 'DAU']
     return (
         <Dropdown
-            title={items[items.map((i) => i.toLowerCase()).indexOf(props.math)] || 'Total'}
-            buttonClassName="btn btn-sm btn-light"
-            style={{ marginLeft: 16 }}
-            data-attr={'math-selector-' + props.index}
+            title={MATHS[props.math || 'total']?.name}
+            buttonClassName="btn btn-sm btn-light ml-2"
+            data-attr={`math-selector-${props.index}`}
         >
-            <Tooltip
-                placement="right"
-                title="Total shows total event volume. If a user performs an event 3 times on one day it counts as 3."
-            >
-                <a href="#" className="dropdown-item" onClick={() => props.onMathSelect(props.index, 'total')}>
-                    Total
-                </a>
-            </Tooltip>
+            {Object.entries(MATHS).map(([key, value]) => (
+                <Tooltip placement="right" title={value.description} key={`math-${key}`}>
+                    <a
+                        href="#"
+                        className="dropdown-item"
+                        onClick={() => props.onMathSelect(props.index, key)}
+                        data-attr={`math-${key}-${props.index}`}
+                    >
+                        {value.name}
+                    </a>
+                </Tooltip>
+            ))}
+        </Dropdown>
+    )
+}
 
-            <Tooltip
-                placement="right"
-                title="Daily Active Users. Selecting DAU will mean a user performing an event 3 times on one day counts as 1."
-            >
-                <a
-                    href="#"
-                    className="dropdown-item"
-                    onClick={() => props.onMathSelect(props.index, 'dau')}
-                    data-attr={'dau-option-' + props.index}
-                >
-                    DAU
-                </a>
-            </Tooltip>
+function MathPropertySelector(props) {
+    return (
+        <Dropdown
+            title={props.mathProperty || 'Select property'}
+            buttonClassName="btn btn-sm btn-light ml-2"
+            data-attr={`math-property-selector-${props.index}`}
+        >
+            {props.properties
+                .filter(({ value }) => value[0] !== '$' && value !== 'distinct_id')
```

Comment:
we should filter for the property type here. i'm guessing the common use case in custom properties will actually be numerical but it will be more comprehensive and intuitive if we remove properties that might be booleans for example so it won't look like you can average a boolean

Reply:
I considered this at first, but then there is no typing in properties, so we need to gracefully handle a situation like: user releases their software with an oversight where a string is accidentally sent in place of a number, user pushes a hotfix, but 100 events with that string remain – in this case we still should be able to count all the other, proper events. So properties that *might* be booleans should be shown. After a bit of rethinking though, it'd be good to hide properties that *never* are numbers (to date). I'll put that in.